modules/aggregation-layers/src/aggregation-layer-v9/cpu-aggregator/cpu-aggregator.ts

```diff
@@ -0,0 +1,189 @@
+import type {Aggregator, AggregationProps} from '../aggregator';
+import {_deepEqual as deepEqual, BinaryAttribute} from '@deck.gl/core';
+import {sortBins, packBinIds} from './sort';
+import {aggregateChannel} from './aggregate';
+import {VertexAccessor, evaluateVertexAccessor} from './vertex-accessor';
+
+/** Settings used to construct a new CPUAggregator */
+export type CPUAggregatorSettings = {
```

Comment:
This mix of `...Settings` and `...Props` here kind of trips me up. 
- Can it be avoided? 
- Or can we work on the naming to make it clearer? 
- Could `Props` or `Options` be used as constructor arguments, rather than `Settings`. The latter seems more unfamiliar to me, but maybe used more extensively in deck.gl than I realize.